onnxruntime/core/framework/provider_bridge_ort.cc

```diff
@@ -7,6 +7,7 @@
 #include "core/framework/data_types.h"
 #include "core/framework/allocatormgr.h"
 #include "core/providers/dnnl/dnnl_provider_factory.h"
+#include "core/session/inference_session.h"
```

Comment:
includes should be in alphabetical order, per https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
In our .clang-format file https://github.com/microsoft/onnxruntime/blob/master/.clang-format#L8, SortIncludes is set to false.

Reply:
There was some push back around that setting when we first added the .clang-format file. Personally I prefer them sorted but it's not something we enforce. 

---
In reply to: [432818969](https://github.com/microsoft/onnxruntime/pull/4091#discussion_r432818969) [](ancestors = 432818969)

Reply:
Ok. Sorry if it sounds like I'm nitpicking here, but I try to follow guidance here: https://github.com/microsoft/onnxruntime/blob/master/docs/Coding_Conventions_and_Standards.md
It doesn't mention any exceptions for that section of the Google style guide.